src/early-late-bound.md

```diff
@@ -1,46 +1,13 @@
 # Early and Late Bound Variables
 
-A generic type may be [_universally_ or _existentially_ quantified][quant]. For
-example,
-
-[quant]: ./appendix/background.md#quantified
-
-```rust,ignore
-fn foo<T>()
-```
-This function claims that the function is well-typed for all types `T`. To use
-the chalk notation: `forall<T> foo<T>`.
-
-Another example:
-
-```rust,ignore
-fn foo<'a>(_: &'a usize)
-```
-This function claims that there is some lifetime `'a` (determined by the
-caller) such that it is well-typed.
-
-One more example:
-
-```rust,ignore
-fn foo<F>()
-where for<'a> F: Fn(&'a u8),
-```
-This function claims that for all lifetimes `'a` and types `F` satisfying the
-bound, it is well-typed:
-
-```txt
-forall<'a, F> {
-    if F: Fn(&'a u8) {
-        foo<'a, f>
-    }
-}
-```
-
+In Rust, generic types are generally [_universally_ quantified][quant].
 Notice, however, that in Rust, we don't have (at the language level)
```

Comment:
Oops, sorry @mark-i-m for the delay -- but this doesn't seem right to me. The type `for<'a> fn(&'a u32)`, for example, is a universally quantified type, as is `dyn for<'a> Foo<&'a u32>` or something like that. It is true that the quantifiers are attached to specific kinds of types (`fn` types, `dyn` types), and we don't have some general purpose "type quantification" mechanism. 

(As far as early- vs late- splits go, any region that is bound in a type is late-bound.)

Reply:
Hi @nikomatsakis, I adjusted this paragraph again in the latest commit, but I'm not sure if I'm just misunderstanding you...